lessons/11-basic-data-structs/assignments/account-generator-cont.md

```diff
@@ -0,0 +1,26 @@
+# Student Account Generator Continued
+
+Now that we know about hashes, we will refactor your original account generator code to utilize arrays with hashes. Since each student has three pieces of data, we will utilize a hash to store these three pieces of data, related to one another.
```

Comment:
I would add something about why this is a better solution. Something kind of like: http://docstore.mik.ua/orelly/perl4/prog/ch09_03.htm